Gruntfile.js

```diff
@@ -392,7 +392,10 @@ module.exports = function (grunt) {
         // version number, but setuptools will "normalize" it to
         // "foobar-0.8.1.dev0", so we need to do the same in order to install
         // the package created by the grunt package task.
-        pyversion = version.replace("-dev", ".dev0");
+        pyversion = version;
+        if (!grunt.file.exists("sdist/tangelo-" + pyversion + zipExt)) {
+            pyversion = version.replace("-dev", ".dev0");
+        }
 
```

Comment:
@manthey do you mind commenting this change, since it's not obvious why it was made?